### DISCUSSION TITLE
Is "C-style comments" accurate, given no support for multiline?

### DISCUSSION THREAD

**Discussion by NReilingh** [cuelang/cue#858] [General]
When I read the CUE tutorial, I took note of the "C-style comments" item listed at https://cuelang.org/docs/tutorials/tour/intro/json/, and interpreted that to mean support for both single-line (`// comment`) and multiline (`/* comment */`) comments.

I was rather surprised, then, when CUE blew up after I tried to feed it some multiline comments. It seems as though only the `//` style is supported.

Is it accurate to call these C-style comments in the tutorial, especially given that C's original comment style was the multiline variation, and the single-line `//` style was actually an extension from C++?

**Comment by myitcv** [comment]
I suspect there might have been a time when CUE _did_ support multi-line comments; @mpvl will be able to confirm.

Either way, they are not support today. So @NReilingh I'd agree - we should make the tutorial line up with [the spec](https://tip.cuelang.org/docs/references/spec/#comments).

**Reply by NReilingh** [reply]
Cool, I've submitted a PR: https://github.com/cuelang/cue/pull/864

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/858.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
